pkg/operator/cluster/ceph/mon/mon.go

```diff
@@ -111,18 +91,11 @@ func New(context *clusterd.Context, namespace, dataDirHostPath, version string,
 		Namespace:           namespace,
 		Version:             version,
 		Size:                size,
-		maxMonID:            -1,
 		waitForStart:        true,
 		monPodRetryInterval: 6 * time.Second,
-		monPodTimeout:       5 * time.Minute,
```

Comment:
these are needed by the mon-health check, will you be refactoring that as well?

Reply:
That will be added again when I got feedback on how we are going to handle health checking with StatefulSet Pods.